packaging/rpm/crc.spec.in

```diff
@@ -60,7 +60,7 @@ export GOFLAGS="-mod=vendor"
 mkdir embed-files
 cp /usr/bin/crc-driver-libvirt embed-files
 cp /usr/bin/crc-admin-helper embed-files/crc-admin-helper-linux
-make GO_EXTRA_LDFLAGS="-B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \n')" GO_EXTRA_BUILDFLAGS="" CUSTOM_EMBED=true EMBED_DOWNLOAD_DIR=embed-files/ release
+make COMMIT_SHA=__COMMIT_SHA__ GO_EXTRA_LDFLAGS="-B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \n')" GO_EXTRA_BUILDFLAGS="" CUSTOM_EMBED=true EMBED_DOWNLOAD_DIR=embed-files/ release
```

Comment:
this means before every release we need to update the spec file in the dist-gist with the `COMMIT_SHA` override. We do need to add that in our rpm creation document, better to have internal CI job for creating this release rpm.

Reply:
We already need to update the spec file to change the version number, add commit logs, ... But yes with this change, it's harder to make manual changes to the .spec file.
An alternative approach which is more complex: we generate our own crc source tarball at release time through github actions with the correct spec file in it, or with an autogenerated file containing the hash that the .spec file should be using.